scripts/check_doc_links.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
```

Comment:
Missing shebang?

Reply:
It was the python commands in the tox file, although i dont know why the other do not require it.